Flake: Memdb tests

```
--- FAIL: TestConcurrentWritePanic (1.00s)
    memdb_test.go:48: 
        	Error Trace:	/home/runner/work/spicedb/spicedb/internal/datastore/memdb/memdb_test.go:48
        	Error:      	Condition never satisfied
        	Test:       	TestConcurrentWritePanic
```

--- a/internal/datastore/memdb/memdb_test.go
+++ b/internal/datastore/memdb/memdb_test.go
@@ -76,7 +76,7 @@ func TestConcurrentWritePanic(t *testing.T) {
 			return g.Wait()
 		})
 		return numPanics > 0
-	}, 1*time.Second, 10*time.Millisecond)
+	}, 3*time.Second, 10*time.Millisecond)
 	require.ErrorIs(err, recoverErr)
 }
 

